source/content/drupal-10.md

```diff
@@ -12,15 +12,31 @@ product: [composer]
 integration: [--]
 ---
 
-Drupal 10 is coming in December 2022! We've been busy conducting early testing on the Pantheon platform, and have developed a process for creating a new Drupal 10 site on our platform.
+Drupal 10 is coming in December 2022! This page will be updated frequently with the latest information and instructions for Drupal 10 on Pantheon.,
+
+## Current Status of Drupal 10 at Pantheon
+
+| Scenario | Status | Documentation |
+|---|---|---|
+| Create a Drupal 10 site on Pantheon | Available | [Create a Drupal 10 Site](/drupal-10#create-a-drupal-10-site)|
+| Upgrade a Drupal 9 site to Drupal 10 manually | Available | [Upgrade from Drupal 9 to Drupal 10](/drupal-10#upgrade-a-drupal-9-site-to-drupal-10) |
+| Keep Drupal 9 site when Drupal 10 is released | Available | |
+| Create Drupal 10 site from Pantheon Dashboard | Planned availability January 2023 | |
+| Upgrade a Drupal 9 site to Drupal 10 using the [Terminus Conversion Tools plugin](https://github.com/pantheon-systems/terminus-conversion-tools-plugin) | Early Access | Use the following command: `terminus conversion:upgrade-d10` |
+| Upgrade a Drupal 9 site to Drupal 10 via dashboard | Not Available or Planned | |
```

Comment:
Let's remove this one for now.

Reply:
Agree. We don't want to talk about "upgrading" yet.